web-app/js/portal/utils/trackUsage.js

```diff
@@ -34,3 +34,11 @@ function trackFiltersUsage(actionKey, label, collection) {
     );
 }
 
+function trackDownloadUsage(actionKey, collection, downloadParams) {
+    trackUsage(
+        OpenLayers.i18n('downloadTrackingCategory'),
+        OpenLayers.i18n(actionKey),
```

Comment:
This doesn't need to load from the dictionary as we've already done that. So you can rename the parameter just to 'action' (as it isn't a key) and remove this call to i18n().